### ISSUE TITLE
phpcs triggers errors with warning

### ISSUE THREAD

**Issue by tristanbes** [staabm/annotate-pull-request-from-checkstyle#13]
Hello,

`bin/phpcs --report=checkstyle --runtime-set ignore_warnings_on_exit 1 | cs2pr`

It generates the output (only warnings): 
```
##[warning]Always use identical comparison unless you need type juggling
##[warning]Always use identical comparison unless you need type juggling
##[warning]Always use identical comparison unless you need type juggling
##[warning]Always use identical comparison unless you need type juggling
##[warning]Always use identical comparison unless you need type juggling
##[warning]Member variable "commonNlv3Manager" contains numbers but this is discouraged
##[error]Process completed with exit code 1.
```

Warning should be ignored because of the flag `--runtime-set ignore_warnings_on_exit 1`

I'm not really sure if it's a bug with cs2pr or with phpcs itself combined with the report `checkstyle`

**Comment by tristanbes** [comment]
Workaround with:
`bin/phpcs --warning-severity=0 --report=checkstyle | cs2pr`

**Comment by korelstar** [comment]
I second the request. The workaround is not sufficient, because warnings are not shown anymore.

It would be nice, if `cs2pr` has a command-line parameter for changing the exit behavior. I.e., if the parameter is given and there were warnings only (no errors), then exit code is `0` -- exit code `1` is then used only if there were errors.

Can this issue please be re-opened or should I open a new issue?

**Comment by staabm** [comment]
PR welcome
